CHANGELOG.md

```diff
@@ -130,6 +134,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Now we can process multiple time constraints ([#1124](https://github.com/Plutonomicon/cardano-transaction-lib/issues/1124))
 - E2E test suite didn't apply settings archive CLI option properly ([#1254](https://github.com/Plutonomicon/cardano-transaction-lib/issues/1254))
 
+### Runtime Dependencies
```

Comment:
Plutip?

Reply:
Do you consider Plutip a runtime dependency or a testing framework?

Happy to add it in but then should we also add BPI and cardano-wallet? As they're not direct dependencies of the Plutip branch?

Reply:
Plutip is a runtime dependency for tests. Test suite is a part of our API.

BPI and cardano-wallet are not our direct dependencies. I will make plutip on our branch specify correct deps for itself soon, without the need for us to override on CTL side, so it will not be a problem anymore.

Reply:
done